common/changes/@microsoft/set-webpack-public-path-plugin/ianc-set-public-path-postprocess_2017-08-02-08-01.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/set-webpack-public-path-plugin",
+      "comment": "Added a new getPostProcessScript configuration option to allow consumers to do arbitrary post-processing on the webpack public path variable.",
```

Comment:
(use the present tense)

---
In reply to: [131004654](https://github.com/Microsoft/web-build-tools/pull/291#discussion_r131004654) [](ancestors = 131004654)